src/components/Editor.vue

```diff
@@ -667,7 +669,13 @@ export default {
 		},
 
 		onSave() {
-			emit('files:file:updated', { fileid: this.fileId })
+			const node = new File({
+				id: this.fileId,
+				source: generateRemoteUrl(`dav/files/${this.currentSession.userId}${this.relativePath}`),
+				mtime: new Date(),
+				mime: this.mime,
```

Comment:
@luka-nextcloud Can you explain why we need to also emit source and mime here? I fear source could break that way if we are using text on a public share link.

Reply:
@juliusknorr I have just checked again, source and mime are mandatory.

Reply:
@juliusknorr according to
https://github.com/nextcloud-libraries/nextcloud-files/blob/333c1fc5f64904ec73dff627118b9872992c0d28/lib/files/nodeData.ts#L80-L82
https://github.com/nextcloud-libraries/nextcloud-files/blob/333c1fc5f64904ec73dff627118b9872992c0d28/lib/files/nodeData.ts#L107-L110